src/mock.user.data.tsx

```diff
@@ -0,0 +1,37 @@
+export const mockUserList = {
+  users: {
+    nodes: [
+      {
+        id: '1',
+        name: 'Allan Lima',
+        email: 'allan@mail.com',
+      },
+      {
```

Comment:
Ainda precisa de usuários mockados? Agora já não está integrado?

Reply:
Como essa parte de usuários ainda está em desenvolvimento, fiquei com receio de apagar caso necessitasse refazer alguns testes com menos usuários.

Reply:
Temos que ter em mente que, num projeto, o código que fica no git é o que todos os devs vêem e dão manutenção. Assim, tudo o que é comentários, logs, códigos provisórios ou de testes não precisam subir nele, O ideal é que aqui tenha apenas o código necessário mesmo, porque senão corremos o risco de ir acumulando código "inútil" com o tempo.

Tem algumas alternativas pra você "manter" esse código sem precisar subi-lo no git. Vc pode basicamente apagar e consultar o histórico aqui pra obter de volta, caso precise, ou vc pode deixar ele ai no sue projeto mesmo, mas não comittar, ou ainda guardar esse código em outro lugar na sua máquina.

Reply:
Entendi! Faz sentido